### PR TITLE
Only allow dataset table download if data is in datahub

### PR DESCRIPTION
**Pull request by Tamba32**
# What? Why?
For each study, download icon no longer shows if the study is not in the datahub.

Resolves https://github.com/cBioPortal/cbioportal/issues/2703

### CODE REVIEW

**Comment by Tamba32** [comment]
On second thought, I should change some of the code. Currently, I just have some of this code from the legacy project: https://github.com/cBioPortal/cbioportal/pull/1767/files. But I think I should try to remove the jquery and replace it will vanilla js. So I will update this PR.

**Comment by inodb** [comment]
@Tamba32 what about the jQuery here? Should we still update this?

File: src/pages/datasetView/DatasetList.tsx [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import * as React from "react";
 import * as _ from 'lodash';
+import $ from 'jquery';
```
Comment (inodb) [inline src/pages/datasetView/DatasetList.tsx]: @Tamba32 can u remove the unused jquery import here?

Comment (Tamba32) [inline src/pages/datasetView/DatasetList.tsx]: Done

**Pull request merged**